### PR TITLE
Add name change for task ID and add support for name_postfix for task id

### PR DESCRIPTION
**Pull request by barloc**
Hello
2 parts in this PR.
1 part: fix logic with use of the name_prefix
In your code if set empty string to the name_prefix argument, then function uses task_id. IMHO, if we set empty string to the argument, we don't want to use any prefix to pod name (all other types).

2 part: add name_postfix and random_name_postfix_length to the operator. if we want to use ConfigMap with Job(or Pod) ConfigMap must have not random name (without uuid postfix). So such optional arguments are looking not bad :)

### CODE REVIEW

File: airflow_kubernetes_job_operator/kubernetes_job_operator.py [hunk 1]
```diff
@@ -90,6 +92,8 @@ def __init__(
             arguments (List[str], optional): The kubernetes pod arguments. Defaults to None.
             image (str, optional): The kubernetes container image to use. Defaults to None.
             name_prefix (str, optional): The kubernetes resource(s) name prefix. Defaults to (corrected) task_id.
+            name_postfix (str, optional): The postfix for all resource name. Defaults to None.
```
Comment (LamaAni) [inline airflow_kubernetes_job_operator/kubernetes_job_operator.py]:
```suggestion
            name_postfix (str, optional): The kuberntes resource(s) name postfix. Defaults to None.
```

**Comment by LamaAni** [comment]
I wonder if we should add a global config for this. e.g. add_task_id_postfix in the ariflow config. In any case, this is correct.

**Review by LamaAni — APPROVED** [review]
Looks good. Small change.

**Review by LamaAni — CHANGES_REQUESTED** [review]
Sorry the text change is needed.

**Comment by barloc** [comment]
Hi
I fixed description in the py file and added description for this arguments to README.

**Review by LamaAni — APPROVED** [review]

**Comment by LamaAni** [comment]
[Release](https://github.com/LamaAni/KubernetesJobOperator/releases/tag/2.0.6)

**Pull request merged**